cli/src/main.rs

```diff
@@ -37,7 +37,16 @@ fn main() {
 
     setup_panic_hooks();
 
-    let mut session = Session::init().unwrap();
+    let session_result = Session::init()
+        .map_err(|e| ApolloError::from(ErrorDetails::CLIConfigError { msg: e.to_string() }));
```

Comment:
Since this will prevent anyone from using the CLI, we should be as verbose as we can on how they should fix whatever problem happened. Even a `You may need to change permissions or delete the config at ~/.apollo/config.toml` or something to help people recover would be great